src/api/status/public/index.html

```diff
@@ -1224,5 +1224,7 @@ <h6 class="mt-3">Thank you for sharing!</h6>
     <script src="assets/js/demo-sidenav.js"></script>
     <!-- Control Center for Material Dashboard: parallax effects, scripts for the example pages etc -->
     <script src="assets/js/material-dashboard.min.js?v=3.0.0"></script>
+
+    <script src="./script.js"></script>
```

Comment:
script.js is too generic.  Also, this should probably go in some kind of src dir.  @Andrewnt219, what's your suggestion, based on the build log work?

Reply:
it can go in `/public/js/feed/index.js`. I also suggest making an api.js file to export all the `fetch` functions partly because it's hard to remember all the different hosts.

```js
// /public/js/api.js 
// ...or /public/js/api/feed.js

const fetchFeeds = () => fetch(`https://${process.env.API_HOST}/v1/posts/feeds`);
const fetchStatus = () => fetch('/v1/status/status');

```

Reply:
This still needs to happen.  `script.js` isn't going to work.  Please do what @Andrewnt219 suggests.